src/functions/Coverage.ps1

```diff
@@ -287,9 +287,16 @@ function Get-CommandsInFile {
         # In PowerShell 5.0, dynamic keywords for DSC configurations are represented by the DynamicKeywordStatementAst
         # class.  They still trigger breakpoints, but are not a child class of CommandBaseAst anymore.
 
+        # ReturnStatementAst is excluded as it's not behaving consistent.
+        # "return" is not hit in 5.1 but fixed in a later version. Using "return 123" we get hit on 123 but not return.
+        # See https://github.com/pester/Pester/issues/1465#issuecomment-604323645
         $predicate = {
             $args[0] -is [System.Management.Automation.Language.DynamicKeywordStatementAst] -or
-            $args[0] -is [System.Management.Automation.Language.CommandBaseAst]
+            $args[0] -is [System.Management.Automation.Language.CommandBaseAst] -or
+            $args[0] -is [System.Management.Automation.Language.BreakStatementAst] -or
+            $args[0] -is [System.Management.Automation.Language.ContinueStatementAst] -or
+            $args[0] -is [System.Management.Automation.Language.ExitStatementAst] -or
+            $args[0] -is [System.Management.Automation.Language.ThrowStatementAst]
         }
```

Comment:
Only updated PSv5+ predicate as we might remove the redundant predicate for v6.
A potential backport would need to extend both predicates.